PlaceRouteHierFlow/placer/ILP_solver.cpp

```diff
@@ -421,6 +425,8 @@ double ILP_solver::GenerateValidSolution(design& mydesign, SeqPair& curr_sp, PnR
   for (unsigned int i = 0; i < mydesign.Blocks.size(); i++) {
     dead_area -= double(mydesign.Blocks[i][curr_sp.selected[i]].width) * double(mydesign.Blocks[i][curr_sp.selected[i]].height);
   }
+  //calculate norm area
+  area_norm = area * 0.1 / (area - dead_area);
```

Comment:
Area seems to be normalized to dead_area. Why are not we simply normalizing to the sum of constituent block areas?

Reply:
@soneryaldiz (area - dead_area) = sum of block areas; Its just for simplified computation.